.github/workflows/test.yml

```diff
@@ -267,6 +267,35 @@ jobs:
       - name: Integration Tests
         run: pnpm -- turbo run integration-tests --filter=cli
 
+  go_rust_integration:
+    name: Go Integration Tests (with Rust FFI)
+    needs: determine_jobs
+    if: needs.determine_jobs.outputs.go_e2e == 'true'
+    timeout-minutes: 30
+    runs-on: ${{ matrix.os.runner }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - name: ubuntu
+            runner: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/setup-turborepo-environment
+        with:
+          target: ${{ matrix.os.name }}
+          github-token: "${{ secrets.GITHUB_TOKEN }}"
+
+      - name: Cache Prysk
+        id: cache-prysk
+        uses: actions/cache@v3
+        with:
+          path: cli/.cram_env
+          key: prysk-venv-${{ matrix.os.name }}
+
+      - name: Integration Tests
+        run: GO_TAG=rust pnpm -- turbo run integration-tests --filter=cli
```

Comment:
Should we include `GO_TAG` in the integration-tests task definition?

Reply:
Good catch!